scripts/data/gen-nodetime/src/relayer/lib/relayer.ts

```diff
@@ -0,0 +1,433 @@
+				return { id: chain.chainId };
+			}
+			const endpointExistsWithDifferentChainID =
+				this.config.chains &&
+				this.config.chains.find(
+					(x) => x.chainId != chain.chainId && x.rpcAddr == chain.rpcAddr
+				);
+			if (endpointExistsWithDifferentChainID)
+				throw new Error(Errors.endpointExistsWithDifferentChainID);
+
+			const chainExistsWithSameEndpoint =
+				this.config.chains &&
+				this.config.chains.find(
+					(x) => x.chainId == chain.chainId && x.rpcAddr == chain.rpcAddr
+				);
+			if (chainExistsWithSameEndpoint) return { id: chain.chainId };
+			this.config.chains.push(chain);
+			return { id: chain.chainId };
+		} catch (e) {
+			throw new Error(Errors.chainSetupFailed + e);
+		}
+	}
+	public createPath([srcID, dstID, options]: [
+		string,
+		string,
+		ConnectOptions
+	]): Path {
+		try {
+			let path = {
+				id: srcID + "-" + dstID,
+				isLinked: false,
+				src: {
+					chainID: srcID,
+					portID: options.sourcePort,
+				},
+				dst: {
+					chainID: dstID,
+					portID: options.targetPort,
+				},
+			};
+			if (!this.config.paths) {
+				this.config.paths = [];
+			}
+			this.config.paths.push({ path, options });
+			return path;
+		} catch (e) {
+			throw new Error(Errors.pathSetupFailed + e);
+		}
+	}
+	public getPath([id]: [string]): Path {
+		if (this.config.paths) {
+			let path = this.config.paths.find((x) => x.path.id == id);
+			if (path) return path.path;
+		}
+		throw new Error(Errors.pathNotExists);
+	}
+	public listPaths(): Path[] {
+		if (this.config.paths) {
+			let paths = this.config.paths.map((x) => x.path);
+			return paths;
+		}
+		throw new Error(Errors.pathsNotDefined);
+	}
+	public async getDefaultAccount([chainID]: string[]): Promise<Account> {
+		const chain = this.chainById(chainID);
+		if (chain) {
+			let client = await this.getIBCClient(chain);
+			return {
+				address: client.senderAddress,
+			};
+		}
+		throw new Error(Errors.chainNotFound + chainID);
+	}
+	public async getDefaultAccountBalance([chainID]: string[]): Promise<Coin[]> {
+		const chain = this.chainById(chainID);
+		if (chain) {
+			let client = await this.getIBCClient(chain);
+			return await client.query.bank.allBalances(client.senderAddress);
+		}
+		throw new Error(Errors.chainNotFound + chainID);
+	}
+	public async link(paths: string[]): Promise<LinkResponse> {
+		if (this.config.paths) {
+			let response: LinkResponse = {
+				linkedPaths: [],
+				alreadyLinkedPaths: [],
+			};
+			for (let pathName of paths) {
+				const path = this.pathById(pathName);
+				if (path?.path.isLinked) {
+					response.alreadyLinkedPaths.push(pathName);
+					continue;
+				}
+				try {
+					await this.createLink(path);
+					response.linkedPaths.push(pathName);
+				} catch (e) {
+					throw new Error(Errors.pathLinkFailed + pathName + ": " + e);
+				}
+			}
+			return response;
+		}
+		throw new Error(Errors.pathsNotDefined);
+	}
+	public async start(paths: string[]): Promise<StartResponse> {
```

Comment:
I receive a bunch of these logs:
```
Relay 0 packets from wot => spn
Relay 0 packets from spn => wot
Get pending acks on wot
Get pending acks on spn
Relay 0 acks from wot => spn
Relay 0 acks from spn => wot
Timeout 0 packets sent from wot
Timeout 0 packets sent from spn
next heights to relay
Checking if spn has recent header of wot
Checking if wot has recent header of spn
Get pending packets on wot
Get pending packets on spn
Relay 0 packets from wot => spn
Relay 0 packets from spn => wot
Get pending acks on wot
Get pending acks on spn
Get pending packets on wot
Get pending packets on spn
Relay 0 acks from wot => spn
Relay 0 acks from spn => wot
Timeout 0 packets sent from wot
Timeout 0 packets sent from spn
next heights to relay
```

Can we rm all of these noise and only log when txs are actually relayed? If ts-relayer does not support this, we can just filter out these logs with regex in our logger impl.

Reply:
added a filter in ConsoleLogger

Reply:
It seems in the code that it hides only the ones that starts with `Relayer`. I think we should hide all of these logs.